ulfhara.lic

```diff
@@ -122,6 +123,23 @@ class UlfHara
         combo
       when /help any of our wounded you encounter/
         fput("accept task")
+        if DRStats.empath? && !DRSpells.active_spells.include?('Regenerate') && @settings.waggle_sets['healing']['Regenerate']
+          data = @settings.waggle_sets['healing']['Regenerate']
+          check_discern(data, @settings) if data['use_auto_mana']
+          release_cyclics
+          DRC.bput('release spell', 'You let your concentration lapse', "You aren't preparing a spell") unless checkprep == 'None'
+          DRC.bput('release mana', 'You release all', "You aren't harnessing any mana")
```

Comment:
We do this in several places. Consider adding as a method to DRCA.

Reply:
would make it much easier to do

Reply:
Consider it for a future PR, if you're feeling like it :)